bond-plugins-open/src/main/java/com/abiquo/bond/plugins/BackupEvents.java

```diff
@@ -99,4 +100,12 @@ public void linkToVMCache(final Set<String> cache)
     {
         // We don't need the cache so just ignore it;
     }
+
+    @Override
+    public void setRestoreQueue(final LinkedBlockingQueue<VMRestoreStatusList> restoreQueue)
+    {
+        System.out
+            .println("BackupEvents:SampleBackupResultsHandler:setRestoreQueue has been called");
```

Comment:
This method is here just because is a class which implements BackupResultsHandler which have setRestoreQueue function. 

This class is doing nothing that print so it has been changed by logs.